<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Toolbox</class>
 <widget class="QMainWindow" name="GUI_Toolbox">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI_Toolbox</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="Slider_selectHoropter">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>711</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ApplyHoropter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Apply Horopter to light field</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_selectHoropter">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>select Horopter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Horopter">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_cut_X">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>155</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>LF - cut X </string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_cut_Y">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>155</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>LF - cut Y </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cut_X">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cut_Y">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_OpenCVCalibrate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load OpenCV xml file to use calibration</string>
    </property>
    <property name="text">
     <string>openCV_calibrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_OpenCVUndistort">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load OpenCV undist file to use calibration</string>
    </property>
    <property name="text">
     <string>openCV_undistort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_OpenCVCalibrateUndistort">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load OpenCV xml file to use calibration and undistortion</string>
    </property>
    <property name="text">
     <string>openCV calib + undist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ApplyStructureTensor">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply ST Occlusion Handling (local)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_accumulate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>accumulate</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_inner">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>inner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_inner">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_outer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_intens">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_outer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>outer</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_intens">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>intensity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_stopDisp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_stopDisp">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>310</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>stopDisp</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_startDisp">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>310</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>startDisp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_startDisp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ApplyStructureTensor_range">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run 'global shift'</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_lightfieldviewer">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Light Field Viewer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_opencvCalibration">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OpenCV Calibration (Single Camera)</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>21</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_structureTensor">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Structure Tensor Orientation Estimation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_structureTensor_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(Occlusion Handling)</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_lightfieldcamconfigurator">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Light Field Camera Configurator</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
